benchmarks/Benchmarks/CertificatesBenchmark/Benchmarks.swift

```diff
@@ -17,10 +17,21 @@ import Foundation
 
 let benchmarks = {
     Benchmark.defaultConfiguration = .init(
-        metrics: [.mallocCountTotal, .syscalls] + .arc
+        metrics: [
+            .mallocCountTotal,
+            .syscalls,
+            .readSyscalls,
+            .writeSyscalls,
+            .memoryLeaked,
+            .retainCount,
```

Comment:
one of these should be `.releaseCount` surely

Reply:
(also need to rerun the threshold generation, as .releaseCount was gone now)

Reply:
fixed in https://github.com/apple/swift-certificates/pull/125/commits/05943cb0badcfaed46b447696263d6b1e39f3caa